src/mail/mjml/purshase_order.mjml

```diff
@@ -0,0 +1,24 @@
+<mjml>
+  <mj-include path="./partial/header.mjml" />
+  <mj-body background-color="#FFFFFF">
+    <mj-section background-color="#2a5cab" padding="10px 0">
+      <mj-column background-color="#008000">
+        <mj-text>{% load i18n %}</mj-text>
+        <mj-text align="center" color="#FFFFFF" font-size="15px" padding="8px 8px">
+          {%trans "Thank you very much for your purchase."%}
+        </mj-text>
+      </mj-column>
+    </mj-section>
+    <mj-include path="./partial/course_description.mjml" />
+    <mj-section background-color="#FFFFFF" padding="0px 50px">
+      <mj-column width="100%">
+        <mj-include path="./partial/hello.mjml" />
+        <mj-text align="center" font-size="18px" padding="10px 50px">
+          {%trans "Your order has been confirmed."%} 
+          {%trans "You have bought "%} `{{product.title}}` {%trans for a price of%} {{product.price}}{{product.price_currency}}.
```

Comment:
I don't think you need to pass price and price_currency into the template context.
I think the string representation of `product.price` should contain the currency symbol.